<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>435</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>572</width>
    <height>432</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1800</width>
    <height>435</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>1</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resource.qrc">
    <normaloff>:/pictures/pics/logo.jfif</normaloff>:/pictures/pics/logo.jfif</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>390</width>
     <height>390</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item alignment="Qt::AlignRight">
       <widget class="QLabel" name="expr">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>26</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(74, 74, 74);</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QPushButton" name="buttn7">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton {
	background-color: rgb(228, 238, 246);
    color: rgb(0, 0, 0);
	border-radius: 10px;
}
QPushButton:pressed{
	background-color: #a9abac;
	color: rgb(0, 0, 0);
	border-radius: 10px;
}</string>
          </property>
          <property name="text">
           <string>7</string>
          </property>
          <property name="shortcut">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="buttn4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton {
	background-color: rgb(228, 238, 246);
    color: rgb(0, 0, 0);
	border-radius: 10px;
}
QPushButton:pressed{
	background-color: #a9abac;
	color: rgb(0, 0, 0);
	border-radius: 10px;
}</string>
          </property>
          <property name="text">
           <string>4</string>
          </property>
          <property name="shortcut">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="buttn0">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton {
	background-color: rgb(228, 238, 246);
    color: rgb(0, 0, 0);
	border-radius: 10px;
}
QPushButton:pressed{
	background-color: #a9abac;
	color: rgb(0, 0, 0);
	border-radius: 10px;
}</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="shortcut">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QPushButton" name="buttnEqual">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton {
	background-color: rgb(85, 198, 228);
    color: rgb(255, 255, 255);
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: #2a97b0;
	color: rgb(255, 255, 255);
	border-radius: 10px;
}</string>
          </property>
          <property name="text">
           <string>=</string>
          </property>
          <property name="shortcut">
           <string>Enter</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="buttn1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton {
	background-color: rgb(228, 238, 246);
    color: rgb(0, 0, 0);
	border-radius: 10px;
}
QPushButton:pressed{
	background-color: #a9abac;
	color: rgb(0, 0, 0);
	border-radius: 10px;
}</string>
          </property>
          <property name="text">
           <string>1</string>
          </property>
          <property name="shortcut">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="buttn6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton {
	background-color: rgb(228, 238, 246);
    color: rgb(0, 0, 0);
	border-radius: 10px;
}
QPushButton:pressed{
	background-color: #a9abac;
	color: rgb(0, 0, 0);
	border-radius: 10px;
}</string>
          </property>
          <property name="text">
           <string>6</string>
          </property>
          <property name="shortcut">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="buttn5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton {
	background-color: rgb(228, 238, 246);
    color: rgb(0, 0, 0);
	border-radius: 10px;
}
QPushButton:pressed{
	background-color: #a9abac;
	color: rgb(0, 0, 0);
	border-radius: 10px;
}</string>
          </property>
          <property name="text">
           <string>5</string>
          </property>
          <property name="shortcut">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="buttnOpenBr">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton {
	background-color: rgb(85, 198, 228);
    color: rgb(255, 255, 255);
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: #2a97b0;
	color: rgb(255, 255, 255);
	border-radius: 10px;
}</string>
          </property>
          <property name="text">
           <string>(</string>
          </property>
          <property name="shortcut">
           <string>(</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="buttnPlus">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton {
	background-color: rgb(85, 198, 228);
    color: rgb(255, 255, 255);
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: #2a97b0;
	color: rgb(255, 255, 255);
	border-radius: 10px;
}</string>
          </property>
          <property name="text">
           <string>+</string>
          </property>
          <property name="shortcut">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="buttn9">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton {
	background-color: rgb(228, 238, 246);
    color: rgb(0, 0, 0);
	border-radius: 10px;
}
QPushButton:pressed{
	background-color: #a9abac;
	color: rgb(0, 0, 0);
	border-radius: 10px;
}</string>
          </property>
          <property name="text">
           <string>9</string>
          </property>
          <property name="shortcut">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="buttnClosingBr">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton {
	background-color: rgb(85, 198, 228);
    color: rgb(255, 255, 255);
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: #2a97b0;
	color: rgb(255, 255, 255);
	border-radius: 10px;
}</string>
          </property>
          <property name="text">
           <string>)</string>
          </property>
          <property name="shortcut">
           <string>)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="buttnBackspace">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton {
	background-color: rgb(85, 198, 228);
    color: rgb(255, 255, 255);
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: #2a97b0;
	color: rgb(255, 255, 255);
	border-radius: 10px;
}</string>
          </property>
          <property name="text">
           <string>⌫</string>
          </property>
          <property name="shortcut">
           <string>Backspace</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="buttn8">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton {
	background-color: rgb(228, 238, 246);
    color: rgb(0, 0, 0);
	border-radius: 10px;
}
QPushButton:pressed{
	background-color: #a9abac;
	color: rgb(0, 0, 0);
	border-radius: 10px;
}</string>
          </property>
          <property name="text">
           <string>8</string>
          </property>
          <property name="shortcut">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="buttn2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton {
	background-color: rgb(228, 238, 246);
    color: rgb(0, 0, 0);
	border-radius: 10px;
}
QPushButton:pressed{
	background-color: #a9abac;
	color: rgb(0, 0, 0);
	border-radius: 10px;
}</string>
          </property>
          <property name="text">
           <string>2</string>
          </property>
          <property name="shortcut">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="buttnMulti">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton {
	background-color: rgb(85, 198, 228);
    color: rgb(255, 255, 255);
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: #2a97b0;
	color: rgb(255, 255, 255);
	border-radius: 10px;
}</string>
          </property>
          <property name="text">
           <string>*</string>
          </property>
          <property name="shortcut">
           <string>*</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="buttn3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton {
	background-color: rgb(228, 238, 246);
    color: rgb(0, 0, 0);
	border-radius: 10px;
}
QPushButton:pressed{
	background-color: #a9abac;
	color: rgb(0, 0, 0);
	border-radius: 10px;
}</string>
          </property>
          <property name="text">
           <string>3</string>
          </property>
          <property name="shortcut">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QPushButton" name="buttnMinus">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton {
	background-color: rgb(85, 198, 228);
    color: rgb(255, 255, 255);
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: #2a97b0;
	color: rgb(255, 255, 255);
	border-radius: 10px;
}</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
          <property name="shortcut">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QPushButton" name="buttnDiv">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton {
	background-color: rgb(85, 198, 228);
    color: rgb(255, 255, 255);
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: #2a97b0;
	color: rgb(255, 255, 255);
	border-radius: 10px;
}</string>
          </property>
          <property name="text">
           <string>/</string>
          </property>
          <property name="shortcut">
           <string>/</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="buttnDelete">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton {
	background-color: rgb(85, 198, 228);
    color: rgb(255, 255, 255);
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: #2a97b0;
	color: rgb(255, 255, 255);
	border-radius: 10px;
}</string>
          </property>
          <property name="text">
           <string>C</string>
          </property>
          <property name="shortcut">
           <string>Del</string>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QPushButton" name="pushButton_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton {
	background-color: rgb(85, 198, 228);
    color: rgb(255, 255, 255);
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: #2a97b0;
	color: rgb(255, 255, 255);
	border-radius: 10px;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(228, 238, 246);</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
